test/unit/views/payment-method-view.js

```diff
@@ -204,6 +205,80 @@ describe('PaymentMethodView', function () {
     });
   });
 
+  describe('selecting vaulted payment methods', function () {
+    beforeEach(function () {
+      this.client = fake.client();
+      this.model = fake.model();
+      this.sandbox.stub(analytics, 'sendEvent');
+    });
+
+    it('sends analytic event when vaulted card selected', function () {
+      var view = new PaymentMethodView({
+        client: this.client,
+        model: this.model,
+        strings: strings,
+        paymentMethod: {
+          type: 'CreditCard',
+          nonce: 'nonce',
+          vaulted: true,
+          details: {
+            lastFour: 1111
+          }
+        }
+      });
+
+      view._choosePaymentMethod();
+
+      expect(analytics.sendEvent.withArgs(this.client, 'vaulted-card.select')).to.be.calledOnce;
+    });
+
+    it('sends analytic event when vaulted paypal selected', function () {
+      var view = new PaymentMethodView({
+        client: this.client,
+        model: this.model,
+        strings: strings,
+        paymentMethod: {
+          type: 'PayPalAccount',
+          vaulted: true,
+          details: {
+            email: 'test@example.com'
+          }
+        }
+      });
+
+      view._choosePaymentMethod();
+
+      expect(analytics.sendEvent.withArgs(this.client, 'vaulted-paypal.select')).to.be.calledOnce;
+    });
+
+    it('does not send analytic event when no vault payment selected', function () {
+      expect(analytics.sendEvent.withArgs(this.client, 'vaulted-card.select')).to.not.be.called;
```

Comment:
Not a fan of having an assertion without an action preceding it. Looks like a view wasn't even created, so I'm not sure what this is testing.

Reply:
[Fixes here](https://github.com/braintree/braintree-web-drop-in/pull/508/commits/d33aadd655c0e1fc1d6f4d347849dd16e364c992)